specs-go/config.go

```diff
@@ -353,11 +353,11 @@ type DeviceCgroup struct {
 	Access *string `json:"access,omitempty"`
 }
 
-// Seccomp represents syscall restrictions
-type Seccomp struct {
-	DefaultAction Action    `json:"defaultAction"`
-	Architectures []Arch    `json:"architectures"`
-	Syscalls      []Syscall `json:"syscalls,omitempty"`
+// LinuxSeccomp represents syscall restrictions
+type LinuxSeccomp struct {
+	DefaultAction Action         `json:"defaultAction"`
```

Comment:
Fixed in e918daac262554b30644cab48f8a53ea656aa908, along with `Operator` and `Arg`